lib/celluloid/groups/proactor.rb

```diff
@@ -0,0 +1,56 @@
+require 'thread'
+
+module Celluloid
+  class Group
+    class Proactor < Group
```

Comment:
I'd probably call it something that describes what it's actually doing like `Spawner`